app/services/remark_generators/residual_balance_checker.rb

```diff
@@ -0,0 +1,31 @@
+module RemarkGenerators
+  class ResidualBalanceChecker
+    def self.call(assessment)
+      new(assessment).call
+    end
+
+    def initialize(assessment)
+      @assessment = assessment
+    end
+
+    def call
+      populate_remarks if residual_balance?
+    end
+
+    private
+
+    def residual_balance?
+      current_accounts = assessment.capital_items.where(description: 'Current accounts')
+      current_account_balance = current_accounts.map(&:value).sum
+      assessment.capital_summary.assessed_capital > assessment.capital_summary.lower_threshold && current_account_balance.positive?
```

Comment:
Observation, not a blocker 🚧 
Would it be worth splitting out 
```ruby
def capital_exceeds_lower_threshold?
  assessment.capital_summary.assessed_capital > assessment.capital_summary.lower_threshold
end
```
to make a slightly easier to read?

Reply:
Good point. I've made the change.